src/site/stages/build/drupal/api.js

```diff
@@ -26,20 +26,25 @@ function encodeCredentials({ user, password }) {
   return Buffer.from(credentials).toString('base64');
 }
 
-const defaultClientOptions = { verbose: true };
+const defaultClientOptions = { verbose: true, maxParallelRequests: 15 };
 
 function getDrupalClient(buildOptions, clientOptionsArg) {
   const buildArgs = {
     address: buildOptions['drupal-address'],
     user: buildOptions['drupal-user'],
     password: buildOptions['drupal-password'],
+    maxParallelRequests: buildOptions['drupal-max-parallel-requests'],
   };
 
+  const maxParallelRequests =
+    buildArgs.maxParallelRequests ?? defaultClientOptions.maxParallelRequests;
```

Comment:
The overrides here...`buildOptions`, `defaultClientOptions`, `clientOptionsArg`, etc...had my head spinning